rifDatabase/Postgres/psql_scripts/alter_scripts/v4_0_alter_11.sql

```diff
@@ -67,15 +67,564 @@
 * Alter 11: More risk Analysis Enhancements; support for Postgres 10 partitioning
```

Comment:
The relative paths in this go one level too high. They say `../../` when the actual files are in `../`. I ran it successfully by changing them all. The alternative, of course, would be to run this script from one level higher, but I don't think that's what we normally do.

Reply:
It will work in both directories "psql_scripts" and "psql_scripts\alter_scripts" with the correct path top the file